mrp_project/__openerp__.py

```diff
@@ -6,10 +6,11 @@
 {
     "name": "MRP Project Link",
     "summary": "Link production with projects",
-    "version": "8.0.1.1.0",
+    "version": "8.0.1.1.1",
     "depends": [
         "mrp_analytic",
         "project",
+        "project_timesheet",
```

Comment:
This dependency is necessary for enabling inheritance in this method: https://github.com/OCA/manufacture/pull/94/files#diff-8655ff19eb2fb4b321a2742bdcf95e8eR33 ```_create_analytic_entries```